### ISSUE TITLE
Nested `img` within `a` tag in Markdown causes Vite Error

### ISSUE THREAD

**Issue by Kutius** [nolebase/integrations#247]
## vitepress-plugin-inline-link-preview

```md
<a href="/img/es-feature.png" target="_blank"><img src="/img/es-feature.png" /></a>
```
This will cause `Vite` Error : `[plugin:vite:vue] Invalid end tag.`

**Comment by nekomeowww** [comment]
Hi, welcome to Nolebase community!

Thanks for reporting, fixed on #248, please wait our CI/CD to quickly rollout a fix on that.

**Comment by Kutius** [comment]
> Hi, welcome to Nolebase community!
> 
> Thanks for reporting, fixed on #248, please wait our CI/CD to quickly rollout a fix on that.

Also Thx for your attention. ^ ^

**Comment by nekomeowww** [comment]
No problem! My pleasure. 

And the fix was already released, please upgrade to v2.2.2 to give it a try.
